### ISSUE TITLE
Nu Html Validator Error 

### ISSUE THREAD

**Issue by tderflinger** [emotion-js/emotion#2195]
**Current behavior:**

I am using Emotion styled components in my Gatsby project. Then I validated the website with the nu-html validator (https://validator.w3.org/nu/). It returns the following error for each generated style:

{
      "type": "error",
      "lastLine": 1,
      "lastColumn": 4840,
      "firstColumn": 4808,
      "message": "Element “style” not allowed as child of element “div” in this context. (Suppressing further errors from this subtree.)",
      "extract": "1f1k0xt0\"><style data-emotion-css=\"6lsb90\">.css-6",
      "hiliteStart": 10,
      "hiliteLength": 33
    },

Is there a way to change the generation of inline styles?

**Expected behavior:**

I expected the nu-html validation to pass with my emotion styled components project.

**Environment information:**

- `react` version: 16.12
- "@emotion/core": "^10.0.27",
-  "@emotion/styled": "^10.0.27",

**Comment by Andarist** [comment]
You can either ignore this problem as it isn't an actual problem (it's only reported by the validator 🤷‍♂️ ) or you can use the advanced SSR setup: https://emotion.sh/docs/ssr#advanced-approach

**Comment by tderflinger** [comment]
Hello, thanks for your reply. I read in the documentation that with Next.js I can insert the styles in the <head>. I think this approach would help me too. Do you know how to easily do this with Gatsby?

**Comment by Andarist** [comment]
It seems that your question lacks some words - could you rephrase?

**Comment by tderflinger** [comment]
Sorry, I inserted a tag (head) that was removed automatically.

**Comment by Andarist** [comment]
It should be possible to replicate what this PR has tried to implement: https://github.com/gatsbyjs/gatsby/pull/17646/files

**Comment by tderflinger** [comment]
Thanks for this hint!
